test/unit/inflatable_model_test.rb

```diff
@@ -1,11 +0,0 @@
-require 'test_helper'
-
-class InflatableModelTest < ActiveSupport::TestCase
-  test "#initialize should assign to declared attributes from passed in hash" do
-    class InflatableModelTestClass < InflatableModel
-      attr_accessor :foo
-    end
-
-    assert_equal 'bar', InflatableModelTestClass.new(foo: 'bar').foo
```

Comment:
I'm pretty sure that `StatisticsAnnouncementTest` would fail if this were wrong.